src/components/results/shortcircuit/shortcircuit-analysis-result.type.ts

```diff
@@ -40,11 +38,46 @@ export interface SCAResultFault {
     feederResults: SCAResultFaultFeederResult[];
 }
 
-export interface ShortcircuitAnalysisResult {
-    resultUuid: UUID;
-    writeTimeStamp: Date;
-    faults: SCAResultFault[];
-}
+type Pageable = {
+    sort: {
+        sorted: boolean;
+        empty: boolean;
+        unsorted: boolean;
+    };
+    pageNumber: number;
+    pageSize: number;
+    offset: number;
+    paged: boolean;
+    unpaged: boolean;
+};
```

Comment:
you can clearly defined that some fields are available only when `unpaged` and `unsorted` are at true with [conditionnal types](https://www.typescriptlang.org/docs/handbook/2/conditional-types.html) or a combination
```typescript
type Unpaged = {
   readonly unpaged: boolean = true;
}
type Paged = {
    readonly unpaged: boolean = false;
    pageNumber: number;
    pageSize: number;
    offset: number;
}
type Pageable = (Unpaged | Paged) & {
    sort: {
        sorted: boolean;
        empty: boolean;
    };
}
```
or even use nullable sort as indication: `sort?: { empty: boolean },`

Reply:
it's coming from [Spring Data Pageable](https://docs.spring.io/spring-data/commons/docs/current/api/org/springframework/data/domain/Pageable.html), so nothing to do...